### DISCUSSION TITLE
Manage Rule in API Rules

### DISCUSSION THREAD

**Discussion by nkwelleepiengomejr** [pocketbase/pocketbase#5276] [Q&A]
I just wanted to ask a few things about the Manage Rule. I have a key role in user collections. However, it seems that certain actions could only be performed by the super-admin. I am wondering if adding `role = 'admin'` in the manage rule would allow me to configure it in such a way that I could let admin users manage another user from a React Application UI.

**Comment by ganigeorgiev** [comment]
It depends what do you mean by "certain actions" (what actions exactly?). From the help text under the field:

> This API rule gives admin-like permissions to allow fully managing the auth record(s), eg. changing the password without requiring to enter the old one, directly updating the verified state or email, etc.

**Reply by nkwelleepiengomejr** [reply]
It's a great project and I loved it, Recently I used it for a practical interview take-home assignment. A simple backend fronted crud app. About the permission is here's what I want: email are not visible to other users so they should be visible to users whose role key is admin. And yes there was exactly the same line just pasted; below the Manage Rule text input butI thought it's for all the API Rules but I guess I understand. That one was only for Manage Rule.
